### DISCUSSION TITLE
Alpine compatibility

### DISCUSSION THREAD

**Discussion by timsofteng** [mason-org/mason.nvim#239] [Q&A] [answered]
Hello. Will it work with alpine Linux?
It use musl instead of glibc.

**Comment by williamboman** [comment, marked as answer]
Hello! Yes, most packages should work on musl-based systems. There might be some packages that make use of a precompiled binary that has been linked to glibc, this is not something I've explicitly checked for to be honest. In my experience, most published binaries are statically linked.
